docs/doc/15-sql-functions/110-semi-structured-functions/index.md

```diff
@@ -9,6 +9,7 @@ This section provides reference information for the semi-structured data functio
 - [JSON_PRETTY](json-pretty.md)
 - [JSON_TYPEOF](json-typeof.md)
 - [PARSE_JSON](parse-json.md)
+- [FLATTEN](flatten.md)
```

Comment:
flatten is a table function, better to also add a reference to the page of table fuction.